lib/representable/deserializer.rb

```diff
@@ -20,7 +20,13 @@ module Representable
   end
 
   OverwriteOnNil = ->(input, options) do
-    input.nil? ? (SetValue.(input, options); Pipeline::Stop) : input
+    if input.nil?
+      function = options[:binding][:setter] ? Setter : SetValue
```

Comment:
I see, so rewrite the PR to make the pipeline factory pick which OverwriteOnNil function to use?